src/modules/esl-panel-group/core/esl-panel-group.ts

```diff
@@ -321,7 +322,11 @@ export class ESLPanelGroup extends ESLBaseElement {
     // activePanels = currentActivePanels - 1 (hide) + 1 if the event produced by 'before:show'
     const activeNumber = this.$activePanels.length - 1 + Number(selfHandled);
 
-    if (activeNumber < this.currentMinItems) return e.preventDefault();
+    if (activeNumber < this.currentMinItems) {
+      if (this.currentMinItems === 1 || this.currentMinItems === this.$panels.length) return e.preventDefault();
+      this.$panels.find(($panel: ESLPanel) => !$panel.open && $panel.show(this.mergeActionParams({event: e, target: $panel})));
```

Comment:
There are a lot of operations here that are not super light, let's discuss possible optimizations and possibly unifications of the algorithm. I believe there is a way to make the first condition as a consequence of the algoritm